.github/workflows/build-pr-preview.yml

```diff
@@ -38,6 +39,8 @@ jobs:
           files_yaml: |
```

Comment:
[Regarding the actions/checkout used on line 33](https://github.com/hashicorp/web-unified-docs/pull/226/files#diff-8113239fc91b00477c335b103f10bda4aa227c80db5970dda69e745f2c8cecc2R33):

ref: ${{ github.event.pull_request.head.sha }} is not needed on because in tj-actions/changed-files we pass in the `base sha` and `head sha`.